modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/utils/bean/APICreationRequestBean.java

```diff
@@ -254,7 +291,8 @@ public APICreationRequestBean(String apiName,String context,String version,Strin
         this.version = version;
         this.provider = provider;
         resourceBeanList = new ArrayList<APIResourceBean>();
-        resourceBeanList.add(new APIResourceBean("GET", "Application & Application User", "Unlimited", "/*"));
+        resourceBeanList.add(new APIResourceBean("GET", "Application & Application User",
```

Comment:
fixed in commit: https://github.com/wso2/product-apim/commit/0627062046b44fe68813b835ecbad7e50ba3e89b